conduit/fair/data/datasets/acs.py

```diff
@@ -123,15 +137,59 @@ def _employment_filter(data: pd.DataFrame) -> pd.DataFrame:
     ],
     target='ESR',  # employment status recode
     target_transform=lambda x: x == 1,
-    group='DIS',
+    group='DIS',  # disability recode
     preprocess=_employment_filter,
     postprocess=lambda x: np.nan_to_num(x, nan=-1),
 )
+_ACSEmployment = BasicProblem(
+    features=[
+        'AGEP',  # age; for range of values of features please check Appendix B.4 of Retiring Adult:
+        # New Datasets for Fair Machine Learning NeurIPS 2021 paper
+        'SCHL',  # educational attainment
+        'MAR',  # marital status
+        'RELP',  # relationship
+        'DIS',  # disability recode
+        'ESP',  # employment status of parents
+        'CIT',  # citizenship status
+        'MIG',  # mobility status (lived here 1 year ago)
+        'MIL',  # military service
+        'ANC',  # ancestry recode
+        'NATIVITY',  # nativity
+        'DEAR',  # hearing difficulty
+        'DEYE',  # vision difficulty
+        'DREM',  # cognitive difficulty
+        'SEX',  # sex
+    ],
+    target='ESR',
+    target_transform=lambda x: x == 1,
+    group='RAC1P',  # recoded detailed race code
+    preprocess=lambda x: x,
+    postprocess=lambda x: np.nan_to_num(x, nan=-1),
+)
+_ACSIncome = BasicProblem(
+    features=[
+        'AGEP',  # age; for range of values of features please check Appendix B.4 of Retiring Adult:
+        # New Datasets for Fair Machine Learning NeurIPS 2021 paper
+        'COW',
+        'SCHL',  # educational attainment
+        'MAR',  # marital status
+        'OCCP',
+        'POBP',
+        'RELP',
+        'WKHP',
```

Comment:
If anyone knows what these other features mean, please speak up.

Reply:
Features are features; we don't care about their underlying meaning, only their value and the distributions they follow